### PR TITLE
Add a section for cursive text

### PR DESCRIPTION
**Pull request by xfq**
***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 23, 2020, 2:20 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2Fw3c%2Fbp-i18n-specdev%2F9fe14d24ac978be661c2645500236205dd08a6ff%2Findex.html%3FisPreview%3Dtrue)

```
[33m📡 HTTP Error 520:[39m [36mhttps://rawcdn.githack.com/w3c/bp-i18n-specdev/9fe14d24ac978be661c2645500236205dd08a6ff/index.html[39m
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/bp-i18n-specdev%2342.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1431,7 +1432,26 @@ <h5>How to's</h5>
  </section>
  </section>
 </section>  
-  
+
+<section id="cursive" class="subtopic">
+  <h3>Cursive text</h3>
+  <p class="advisement" id="cursive_opacity"><a class="self" href="#cursive_opacity">&#x200B;</a>
+    Overlapping joinings should not be exposed when transparency is applied to the cursive text such as Arabic, Mongolian, and N'Ko.
```
Comment (r12a) [inline index.html]: I suggest tweaking as follows:
"Overlaps should not be exposed when transparency is applied to the joined letters in cursive text, such as for Arabic, Mongolian, and N'Ko."

File: index.html [hunk 2]
```diff
@@ -1431,7 +1432,26 @@ <h5>How to's</h5>
...
+  </p>
+  <p class="advisement" id="cursive_border"><a class="self" href="#cursive_border">&#x200B;</a>When adding text border, a joined letter should not be separated from its joined neighbors in cursive text.</p>
```
Comment (r12a) [inline index.html]: how about 
"When adding a text stroke or shadow, joined letters should not be separated from their neighbors in cursive script text."

File: index.html (hunk 1 above)
Comment (xfq) [reply index.html]: Updated. Thank you!

File: index.html (hunk 2 above)
Comment (xfq) [reply index.html]: Looks good to me. Updated.

**Pull request merged**